unordered_associative_task_1/student/src/impl.cpp

```diff
@@ -1 +1,10 @@
 #include "impl.h"
+#include <unordered_set>
+
+bool is_anagram(const std::string &first, const std::string &second)
+{
+    const std::unordered_multiset<char> firstCharset{first.cbegin(), first.cend()};
```

Comment:
In such case, ordered multiset will be much simpler.

Smart, but hard to understand and non traditional way :)

Not sure if it will work for complex objects (not the char), because order is unknown